decidim-comments/lib/decidim/comments/engine.rb

```diff
@@ -37,6 +37,19 @@ class Engine < ::Rails::Engine
         end
       end
 
+      initializer "decidim_comments.register_icons" do
+        common_parameters = { category: "actions", engine: :comments }
```

Comment:
Let's merge the "action" and "actions" categories please
```suggestion
        common_parameters = { category: "action", engine: :comments }
```